### ISSUE TITLE
New installation broken

### ISSUE THREAD

**Issue by warpech** [jorgegonzalez/zoe#107]
After a fresh install in macOS vis `sudo npm install -g zoe`, I get the following error:

```
$ zoe
Zoe 1.29.0
Error: Failed to load plugin jest: Cannot find module 'eslint-plugin-jest'
Require stack:
- /usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config/plugins.js
- /usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config.js
- /usr/local/lib/node_modules/zoe/node_modules/eslint/lib/cli-engine.js
- /usr/local/lib/node_modules/zoe/node_modules/eslint/lib/api.js
- /usr/local/lib/node_modules/zoe/dist/engine.js
- /usr/local/lib/node_modules/zoe/dist/zoe.js
- /usr/local/lib/node_modules/zoe/dist/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
    at Function.resolve (/usr/local/lib/node_modules/zoe/node_modules/v8-compile-cache/v8-compile-cache.js:166:23)
    at Plugins.load (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config/plugins.js:104:29)
    at Array.forEach (<anonymous>)
    at Plugins.loadAll (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config/plugins.js:165:21)
    at loadFromDisk (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config/config-file.js:529:35)
    at Object.load (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config/config-file.js:587:20)
    at Config.getLocalConfigHierarchy (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config.js:240:44)
    at Config.getConfigHierarchy (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config.js:192:43)
    at Config.getConfigVector (/usr/local/lib/node_modules/zoe/node_modules/eslint/lib/config.js:299:21)
```

This is fixable by running `sudo npm install -g eslint-plugin-jest`

**Comment by jorgegonzalez** [comment]
Hoping to fix this in #104.

**Comment by jorgegonzalez** [comment]
Closed in `zoe@1.31.4`
